src/modules/roc_audio/resampler_reader.h

```diff
@@ -69,7 +71,7 @@ class ResamplerReader : public IReader, public core::NonCopyable<> {
     IReader& reader_;
 
     core::Slice<sample_t> frames_[3];
-    const size_t frame_size_;
+    core::nanoseconds_t frame_length_;
```

Comment:
> Should it be of type core::nanoseconds_t or size_t ? Because in the above commits, I thought it should have been changed to core::nanoseconds_t and changed ResamplerReader.

Hm, I thought you were asking about constructor parameter, but if you were asking about the class field, sorry, I probably misled you.

The constructor parameter should be `nanoseconds_t frame_length` because it accepts the frame duration in time units:

```
    ResamplerReader(...
                    packet::channel_mask_t channel_mask,
                    core::nanoseconds_t frame_length,
                    size_t sample_rate);
```

However the class field likely should be `size_t frame_size_` because it hold the frame size in samples.

Reply:
Yeah that's alright. I will fix them.